backends/src/client/permit.rs

```diff
@@ -572,6 +572,7 @@ pub enum Error {
     #[error("response error: {0}")]
     ResponseError(ResponseContent),
 }
+pub type Result<T> = std::result::Result<T, Error>; 
```

Comment:
This should be closer to the top of the file. And maybe even in the service client file.